files/udev/41-gamepad.rules

```diff
@@ -0,0 +1,9 @@
+# Udev rule for the Logitech F710 controller, shipped with some older robots
+KERNEL=="js*", ATTRS{idVendor}=="046d", ATTRS{idProduct}=="c21f", SYMLINK="input/f710"
+
+# Udev rule for the PS4 controller currently shipped with most robots
+KERNEL=="js*", ATTRS{idVendor}=="8087", ATTRS{idProduct}=="07dc", SYMLINK="input/ps4"
+KERNEL=="js*", ATTRS{idVendor}=="8087", ATTRS{idProduct}=="07da", SYMLINK="input/ps4"
+
+# Alternate Udev rule for the PS4 controller (if it's detected as DualShock 4)
```

Comment:
Rather than a global file for all gamepad-related rules, can we use the 41-playstation.rules and 41-logitech.rules files from the clearpath_iso project?

Reply:
Sure, this was using what was in `jetson_setup` but updating.